addons/web/static/src/env.js

```diff
@@ -168,3 +169,39 @@ async function _startServices(env, toStart) {
         return null;
     }
 }
+
+/**
+ * Create an application with a given component as root and mount it. If no env
+ * is provided, the application will be treated as a "root": an env will be
+ * created and the services will be started, it will also be set as the root
+ * in `__WOWL_DEBUG__`
+ *
+ * @param {import("@odoo/owl").Component} component the component to mount
+ * @param {HTMLElement} target the HTML element in which to mount the app
+ * @param {Object} [param2] options
+ * @param {Object} [param2.env] an optional environment. If none is provided,
+ *  a new one will be created automatically and the services will be started.
+ * @param {string} [param2.name] a name for the application, will show up in the
+ *  Owl devtools
+ */
+export async function mountComponent(component, target, { env, name } = {}) {
+    const isRoot = !env;
+    if (isRoot) {
+        env = await makeEnv();
+        await startServices(env);
+    }
+    const app = new App(component, {
+        env,
+        name,
+        templates,
+        dev: env.debug,
+        warnIfNoStaticProps: true,
+        translatableAttributes: ["data-tooltip"],
+        translateFn: _t,
+    });
+    const root = await app.mount(target);
+    if (isRoot) {
+        odoo.__WOWL_DEBUG__ ||= { root };
```

Comment:
having a `__WOWL_DEBUG__` root set is required for tours to be able to scan barcodes and by some other tour utils, and I don't think we want people to have to worry about setting `__WOWL_DEBUG__` to have working tours in their apps, since it's easy to forget. (it's also used by the clickbot but this is only relevant in the webclient)

Having multiple "root" apps (that have different environments) is currently very difficult because starting the services more than once crashes. I guess you could have 2 independent apps and reuse the env, in that case both of the apps would semantically be "roots", and this helper would only set one of them as root. While this may be a little confusing, since `__WOWL_DEBUG__` is only used to access the env, it works just fine. I think in master we should rename this to `__ENV__` or something, and maybe only set it in test mode.